src/block/icon-list-item/block.json

```
@@ -0,0 +1,17 @@
+{
+	"apiVersion": 3,
+	"name": "stackable/icon-list-item",
+	"title": "Icon List Item",
+	"description": "A new block.",
+	"category": "stackable",
+	"usesContext": [ "postId", "postType", "queryId" , "stackable/innerBlockOrientation", "stackable/ordered", "stackable/uniqueId", "stackable/isIndented" ],
+	"keywords": [
+		"Keywords that are not in the title"
+	],
+	"parent": [
+		"stackable/icon-list"
+	],
+	"textdomain": "stackable-ultimate-gutenberg-blocks",
+	"stk-type": "special",
```

This should not be visible in the settings - this should not be allowed to be disabled
```suggestion
	"stk-type": "hidden",
```